site/gatsby-site/src/templates/cite-dynamic.js

```diff
@@ -0,0 +1,604 @@
+          title,
+          mongodb_id,
+          report_number,
+          isResponse: tags && tags.includes(RESPONSE_TAG),
+        })
+      );
+
+      timelineTemp.push({
+        date_published: incidentTemp.date,
+        title: 'Incident Occurrence',
+        mongodb_id: 0,
+        isOccurrence: true,
+      });
+
+      const variantsTemp = sortedIncidentReports.filter((report) => !isCompleteReport(report));
+
+      setEntities(entities);
+      setMetaTitle(`Incident ${incidentTemp.incident_id}: ${incidentTemp.title}`);
+      setMetaDescription(incidentTemp.description);
+      setMetaImage(image.createCloudinaryURL());
+      setTimeline(timelineTemp);
+      setSortedReports(sortedReports);
+      setVariants(variantsTemp);
+      setIncident(incidentTemp);
+    }
+  }, [incidentData]);
+
+  const [subscribeToNewReportsMutation, { loading: subscribing }] =
+    useMutation(UPSERT_SUBSCRIPTION);
+
+  const subscribeToNewReports = async () => {
+    if (!isSubscribed) {
+      if (isRole('subscriber')) {
+        try {
+          const incidentId = incident.incident_id;
+
+          await subscribeToNewReportsMutation({
+            variables: {
+              query: {
+                type: SUBSCRIPTION_TYPE.incident,
+                userId: { userId: user.id },
+                incident_id: { incident_id: incidentId },
+              },
+              subscription: {
+                type: SUBSCRIPTION_TYPE.incident,
+                userId: {
+                  link: user.id,
+                },
+                incident_id: {
+                  link: incidentId,
+                },
+              },
+            },
+          });
+
+          addToast({
+            message: (
+              <>
+                {t(`You have successfully subscribed to updates on incident {{incidentId}}`, {
+                  incidentId,
+                })}
+              </>
+            ),
+            severity: SEVERITY.success,
+          });
+        } catch (e) {
+          addToast({
+            message: <label>{t(e.error || 'An unknown error has ocurred')}</label>,
+            severity: SEVERITY.danger,
+          });
+        }
+      } else {
+        addToast({
+          message: (
+            <Trans i18n={i18n}>
+              Please <Link to={localizePath({ path: '/login', language: locale })}>log in</Link> to
+              subscribe
+            </Trans>
+          ),
+          severity: SEVERITY.success,
+        });
+      }
+    } else {
+      addToast({
+        message: (
+          <Trans i18n={i18n}>
+            Please go to your{' '}
+            <Link to={localizePath({ path: '/account', language: locale })}>account</Link> to manage
+            subscriptions
+          </Trans>
+        ),
+        severity: SEVERITY.success,
+      });
+    }
+  };
+
+  const incidentResponded = sortedReports.some(
+    (report) => report.tags && report.tags.includes(RESPONSE_TAG)
+  );
+
+  console.log(loading, incident);
```

Comment:
Console log (you didn't ask for review so don't remove this if you still need it)